src/contributors/vineetnilawar.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "Vineet Nilawar", 
+    "desc": "Full Stack Developer",
+    "github" : "https://github.com/vineetnilawar",
```

```suggestion
    "github": "https://github.com/vineetnilawar",
```
Remove unwanted `white-space`.